packages/local-signing-manager/src/lib/local-signing-manager.ts

```diff
@@ -181,6 +196,33 @@ export class LocalSigningManager implements SigningManager {
     return address;
   }
 
+  /**
+   * Add a new Account to the Signing Manager via newly generated mnemonics
+   *
+   * @returns the mnemonic of the added Account along its keyring pair
+   *
+   * @throws if called before calling `setSs58Format`. Normally, `setSs58Format` will be called by the SDK when instantiated.
+   *   If Accounts need to be pre-loaded, it should be done by passing them to the `create` method or you can use the `generate` method to create and pre-load an Account
+   */
+  public createAndAddAccount(): { mnemonic: string; keyringPair: IKeyringPair } {
```

Comment:
My preference would be to expose this as just `createAccount` and leave the user to add it in a different call.

Then a chain connection isn't needed to make an Account, so it seems more flexible to me. If the user wants a signing manager with pre-populated accounts they can use the option in create (or as `generate` method as implemented.)

On the other hand, this seems a bit easier to use for the user, so I am not against it, but I having the function call do one thing seems a bit cleaner to me.

Reply:
Maybe make this `createAccount` as a `static` method so the user can get the mnemonic before instantiating an instance.

Reply:
Could be `generateAccount` too, to make it a bit more clear it's not making state that `create` implies from being used so much in that context.

Reply:
+1 on `generateAccount` vs `createAccount`

I'd be a bit clearer on the docs as well, something like

```
Generate a new Polymesh account and return its mnemonic. This account can be used with the Local Signing Manager to sign transactions via the Polymesh SDK

@note make sure to store the returned mnemonic somewhere safe since it will not be accessible by any means after being returned, and it gives full control over the account
```